frontend/src/pages/ranking/index.tsx

```diff
@@ -150,11 +166,18 @@ export const RankingPage: React.FC = () => {
 
 export const getStaticProps: GetStaticProps = async ({ locale }) => {
   const client = getApolloClient();
-
+  console.log("GetStaticProps ranking");
   await prefetchSharedQueries(client, locale);
+  const {
+    data: {
+      conference: { topics },
+    },
+  } = await prefetchTopics(client, locale);
 
   return addApolloState(client, {
-    props: {},
+    props: {
+      topics: topics,
+    },
```

Comment:
you don't need all of this

We use Apollo cache to get the data, so you only need to call `queryTopics`

Reply:
But why re-execute the query (ok the second time will be cached...) instead of just give the topics to the component with `props`?

Reply:
you don't refetch, the first time it will come from the cache

then you will be able to reuse the same query when the topic changes